parser/src/main/java/com/linkedin/avroutil1/model/SchemaOrRef.java

```diff
@@ -22,30 +22,36 @@ public class SchemaOrRef implements LocatedCode {
      * the FQCN of a referenced schema, if this represents a reference
      */
     private final String ref;
+    /**
+     * the namespace of the parent schema, which is necessary for reference resolution
```

Comment:
"which might be necessary for reference resolution if the ref is not a fullname"